lld/test/ELF/incompatible.s

```diff
@@ -1,3 +1,10 @@
+# REQUIRES: aarch64
+# RUN: rm -rf %t
+# RUN: not ld.lld -o /dev/null %t --irpgo-profile=/dev/null --bp-startup-sort=function --call-graph-ordering-file=/dev/null 2>&1 | FileCheck %s --check-prefix=IRPGO-ERR
```

Comment:
This test is for incompatible relocatable files. Add such tests to bp-section-orderer.s

Please spend more time reading the existing tests to check whether the new tests fit there